bulk-import/ndjson-generator/Helpers/DataGenerator.cs

```diff
@@ -0,0 +1,56 @@
+﻿namespace NDJsonGenerator.Helpers
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Text.Json;
+    using Azure.DigitalTwins.Core.Serialization;
+
+    public class DataGenerator
+    {
+        private static readonly string ModelsTemplate = "{{\"@id\":\"dtmi:com:microsoft:azure:iot:model{0};1\",\"@type\":\"Interface\",\"contents\":[{{\"@type\":\"Property\",\"name\":\"prop0\",\"schema\":\"string\"}},{{\"@type\": \"Component\",\"name\":\"Component1\",\"schema\":\"dtmi:com:microsoft:azure:iot:componentmodel;1\"}},{{\"@type\":\"Property\",\"name\":\"prop1\",\"schema\":\"string\"}},{{\"@type\":\"Relationship\",\"name\":\"has\",\"target\":\"dtmi:com:microsoft:azure:iot:model{1};1\",\"properties\":[{{\"@type\":\"Property\",\"name\":\"relationshipproperty1\",\"schema\":\"string\"}},{{\"@type\":\"Property\",\"name\":\"relationshipproperty2\",\"schema\":\"string\"}}]}}],\"description\":{{\"en\":\"This is the description of model\"}},\"displayName\":{{\"en\":\"This is the display name\"}},\"@context\":\"dtmi:dtdl:context;2\"}}" + Environment.NewLine;
+        private static readonly string ComponentModel = $"{{ \"@id\": \"dtmi:com:microsoft:azure:iot:componentmodel;1\", \"@type\": \"Interface\", \"@context\": \"dtmi:dtdl:context;2\", \"displayName\": \"Component1\", \"contents\": [ {{ \"@type\": \"Property\", \"name\": \"ComponentProp1\", \"schema\": \"string\" }} ] }}" + Environment.NewLine;
+        private static readonly string TwinsTemplate = "{{\"$dtId\":\"twin{0}\",\"$metadata\":{{\"$model\":\"dtmi:com:microsoft:azure:iot:model{1};1\"}},\"prop0\":\"propertyValue0\",\"prop1\":\"propertyValue1\", \"Component1\":{{ \"$metadata\":{{ }}, \"ComponentProp1\": \"value\"}} }}" + Environment.NewLine;
+        private static readonly string RelationshipsTemplate = "{{\"$dtId\":\"twin{0}\",\"$relationshipId\":\"relationship\",\"$targetId\":\"twin{1}\",\"$relationshipName\":\"has\",\"relationshipProperty1\":\"propertyValue1\",\"relationshipProperty2\":\"propertyValue2\"}}" + Environment.NewLine;
+
+        public static IEnumerable<string> GenerateModels(int numModels)
+        {
+            List<string> models = new List<string>();
+
+            // Add Component Model
+            models.Add(ComponentModel);
+            for (var i = 0; i < numModels; i++)
+            {
+                string model = string.Format(ModelsTemplate, i, (i + 1) % numModels);
```

Comment:
Considering using `yield return string.Format(...)` instead of defining an empty list and adding incrementally.

Reply:
(same applies to other functions)